compare50/_data.py

```diff
@@ -112,17 +112,23 @@ class Submission:
     Represents a single submission. Submissions may either be single files or
     directories containing many files.
     """
-    _store = IdStore(key=lambda sub: (sub.path, sub.files))
+    _store = IdStore(key=lambda sub: (sub.path, sub.files, sub.large_files, sub.undecodable_files))
 
     path = attr.ib(converter=pathlib.Path, cmp=False)
     files = attr.ib(cmp=False)
+    large_files = attr.ib(default=attr.Factory(tuple), cmp=False, repr=False)
+    undecodable_files = attr.ib(default=attr.Factory(tuple), cmp=False, repr=False)
     preprocessor = attr.ib(default=lambda tokens: tokens, cmp=False, repr=False)
     is_archive = attr.ib(default=False, cmp=False)
     id = attr.ib(init=False)
 
     def __attrs_post_init__(self):
         object.__setattr__(self, "files", tuple(
             [File(pathlib.Path(path), self) for path in self.files]))
+        object.__setattr__(self, "large_files", tuple(
+            [File(pathlib.Path(path), self) for path in self.large_files]))
```

Comment:
Do we actually need to create Ids for these and such? I.e. any reason to actually make these File objects?

Reply:
In fact I think this could have performance implications since it potentially bumps up the size of the matrix in winnowing. Maybe. idk.

Reply:
Shit, you're probably right. Good call!